packages/image_picker/image_picker_for_web/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
-## 3.0.2
+## 3.0.3
+
+* Migrates package and tests to `platform:web`.
+* Updates minimum supported SDK version to Flutter 3.16.0/Dart 3.2.0.
 
+## 3.0.2
```

Comment:
there should be an empty line after this one.

Reply:
Fixed!